modules/keepalived-limitation-for-ipi.adoc

```diff
@@ -0,0 +1,48 @@
+[discrete]
+[id="keepalived-limitation-for-ipi_{context}"]
+= Identifying pre-existing virtual routers
+
+With installer-provisioned infrastructure (IPI) installations, the Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have pre-existing virtual routers using the standard IPv4 or IPv6 multicast groups, you should check to see which VIPs the installation will choose to avoid conflicts.
```

Comment:
The Openshift cluster Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router-id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have multiple clusters using the same network, you should check to see which VIPs the installation will choose to avoid conflicts.

@eslutsky 
@janosdebugs 
What do you think? I tried to simplify it a bit since the explanation is scary (I think)

Reply:
Done.